### ISSUE TITLE
Audio voiceovers continue to play when quickly dismissing the audio bar

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#4580]
**Describe the bug**
It's possible for audio to continue playing even when the audio bar is hidden.

**To Reproduce**
1. Open a lesson that has voiceovers.
2. Click the headphone icon to bring up the audio bar.
3. Right before autoplay starts, click the headphone icon to collapse the audio bar.
4. Observe that audio still starts playing despite the bar no longer being visibile.

**Expected behavior**
Audio should never play when the bar is hidden.

**Demonstration**
A screenshot can't really capture this, and I'm not sure if a video would include the audio so I decided not to.

**Environment**
- Device/emulator being used: Nexus 5X
- Android or SDK version (e.g. Android 5 or SDK 21): SDK 27
- App version (you can get this through system app settings or via the admin controls menu in-app): 0.9-beta-3f935261e0.

**Additional context**
None.

**Comment by BenHenning** [comment]
Duplicate of #1801.
